Sources/Shared/Request+Teams.swift

```diff
@@ -28,15 +28,34 @@ public extension Request {
 
     /// Returns a new request to fetch an array of team members.
     /// - Parameter uri: The team member's URI.
+    /// - Parameter hasAccessOnly: Flag that determines the set of players to be returned. When `true` it returns only
+    /// the list of players added to the folder. When `false` returns all team members.
     /// - Parameter filterString: The filter string to be applied to the fields request parameter.
-    static func getTeamMembers(forURI uri: String, filterString: String? = nil) -> Request {
-
-        let path = filterString.map { "\(uri)?\(String.fieldsKey)=\($0)" } ?? uri
-
+    static func getTeamMembers(
+        forURI uri: String,
+        hasAccessOnly: Bool? = nil,
+        filterString: String? = nil
+    ) -> Request {
+        let hasAccessOnlyPair = hasAccessOnly.map { "\(String.hasAccessOnly)=\($0)" }
+        let filterStringPair = filterString.map { "\(String.fieldsKey)=\($0)" }
+        let parametersString = [
+            hasAccessOnlyPair,
+            filterStringPair
+        ]
+        .compactMap { $0 }
+        .joined(separator: "&" )
+        let path = [
+            uri,
+            parametersString
+        ]
+        .joined(separator: "?")
         return Request(path: path)
     }
 }
 
+// MARK: - Constants
+
 private extension String {
+    static let hasAccessOnly = "has_access_only"
```

Comment:
Nit: Do we not want to append `Key` to this constant as well to stay consistent?